prow/cmd/hook/main.go

```diff
@@ -56,6 +57,7 @@ var (
 	githubBotName     = flag.String("github-bot-name", "", "Name of the GitHub bot.")
 	webhookSecretFile = flag.String("hmac-secret-file", "/etc/webhook/hmac", "Path to the file containing the GitHub HMAC secret.")
 	githubTokenFile   = flag.String("github-token-file", "/etc/github/oauth", "Path to the file containing the GitHub OAuth secret.")
+	slackTokenFile    = flag.String("slack-token-file", "/etc/slack/token", "Path to the file containing the Slack Kubernetes Team Token.")
```

Comment:
I prefer to keep a default value and test existence of the file later part of the code?